src/Polyester.jl

```diff
@@ -25,6 +26,21 @@ export batch, @batch, disable_polyester_threads
 include("batch.jl")
 include("closure.jl")
 
+
+# see https://github.com/JuliaSIMD/Polyester.jl/issues/30
+"""
+    Polyester.reset_threads!()
+
+Resets the threads used by [Polyester.jl](https://github.com/JuliaSIMD/Polyester.jl).
+"""
+function reset_threads!()
+  PolyesterWeave.reset_workers!()
+  for i in 1:(PolyesterWeave.dynamic_thread_count() - 1)
+      ThreadingUtilities.initialize_task(i)
```

Comment:
I would only reinitialize if not running.

Reply:
As implemented in 514b7ac?